pkgs/development/python-modules/pastescript/default.nix

```diff
@@ -33,10 +34,7 @@ buildPythonPackage rec {
   # test suite seems to unset PYTHONPATH
   doCheck = false;
 
-  nativeCheckInputs = [
-    nose
-    pytestCheckHook
-  ];
+  nativeCheckInputs = [ pytestCheckHook ];
```

Comment:
The tests are still broken, I guess?

Reply:
Yes. The package exports its namespace as `paste` which clobbers the pre-existing dependency with the same name.

Reply:
pastedeploy does something similar but its tests pass. Probably the use of the deprecated `pkg_resources` is causing the issue? Anyways, it's not something we should try to fix.